src/cascade/input_data/db/__init__.py

```diff
@@ -1,3 +1,13 @@
+try:
+    import db_queries
+except ImportError:
+
+    class DummyDBQueries:
+        def __getattr__(self, name):
+            raise ImportError(f"Required package db_queries not found")
+
+    db_queries = DummyDBQueries()
```

Comment:
Huh. Interesting. This isn't a fake, a mock, or a stub. It's more of an explode-on-impact.

Reply:
Hey, could we do this for `db_tools` and `save_results` in the same place?